### PR TITLE
remove symbol search

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: process.ts [hunk 1]
```diff
@@ -495,23 +494,35 @@ async function taskLoadModule(
...
     "color:cyan",
     "color:none",
   );
-  const requests: MultipleBatchRequest[] = [];
-  requests.push(moduleToRequest(module));
-  await loadDocNodes(requests, module, version);
-  await upload(requests);
+
+  moduleIndex ??= (await getDenoLandApp()).initIndex(indexes.MODULE_INDEX);
+  await moduleIndex.saveObject({
```
Comment (bartlomieju) [inline process.ts:511]: Why `saveObject` instead of update?

File: auth.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ import {
   objectSetKey,
   objectToEntity,
 } from "google_datastore";
-import { config } from "std/dotenv/mod.ts";
+import { load } from "std/dotenv/mod.ts";
```
Comment (bartlomieju) [inline auth.ts:17]: Seems like an unrelated change

Comment (crowlKats) [reply auth.ts:17]: its to fix ci; mishap when updating std a few days ago

File: process.ts (hunk 1 above)
Comment (crowlKats) [reply process.ts:511]: there is no updateObject, instead saveObject overwrites if there is an existing one

**Review by bartlomieju — APPROVED** [review]
LGTM
